src/components/SignUp/SignUpInfo.tsx

```diff
@@ -27,24 +29,68 @@ const SignUpInfoButton = styled(Button)({
     "&:hover": {
         backgroundColor: dark,
     }
-})
+});
+
+const validationSchema = yup.object({
+    email: yup
+      .string('Enter your email')
+      .email('Enter a valid email')
+      .required('Email is required'),
+    password: yup
```

Comment:
the password check do not meet ticket specification

Reply:
Modified according to card contents